entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/model/SolrPlace.java

```diff
@@ -0,0 +1,168 @@
+package eu.europeana.entitymanagement.solr.model;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.solr.client.solrj.beans.Field;
+
+import eu.europeana.entitymanagement.definitions.model.Place;
+import eu.europeana.entitymanagement.vocabulary.AgentSolrFields;
+import eu.europeana.entitymanagement.vocabulary.EntitySolrFields;
+import eu.europeana.entitymanagement.vocabulary.PlaceSolrFields;
+
+/*
+ * TODO:see how to save the wasPresentAt, referencedWebResource, isAggregatedBy, and entityIdentifier fields
+ */
+public class SolrPlace extends Place {
+	
+	private String payload;
+
+	public SolrPlace() {
+		super();
+	}
+
+	public SolrPlace(Place place) {
+		super();
```

Comment:
use super(place) and remove the use of redundant setters

Reply:
Done.